provision-gcc-VM.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -eux
+
+usage() {
+  cat <<EOF
+  usage: $0 -a VM_IP_ADDRESS -p VM_ROOT_PASSWORD
+
+  This script installs the required software and configures the VM,
+  described by its VM_IP_ADDRESS passed as argument, for installing
+  omnibus manually omnibus gem and be able to build codethink-toolchain
```

Comment:
"for installing omnibus manually omnibus gem" -- something doesn't work here for me.

Reply:
"for manually installing omnibus gem" reads better.